Src/DependencyCollector/Shared/Implementation/ProfilerHttpProcessing.cs

```diff
@@ -301,9 +301,16 @@ private void OnEnd(object exception, object thisObj, object returnValue)
                 {
                     this.TelemetryTable.Remove(thisObj);
                     DependencyTelemetry telemetry = telemetryTuple.Item1;
-                    
+
                     var responseObj = returnValue as HttpWebResponse;
 
+                    var webException = exception as WebException;
```

Comment:
What if responseObj is already set at this point (after previous line)? Should we be overwriting it? Or perhaps only take it from WebException if not already set?